lib/multiplexer/multiplexer.go

```diff
@@ -310,6 +340,22 @@ var (
 	tlsPrefix   = []byte{0x16}
 )
 
+// This section defines Postgres wire protocol messages detected by Teleport:
+//
+// https://www.postgresql.org/docs/13/protocol-message-formats.html
+var (
+	// postgresSSLRequest is always sent first by a Postgres client (e.g. psql)
+	// to check whether the server supports TLS.
+	postgresSSLRequest = []byte{0x0, 0x0, 0x0, 0x8, 0x4, 0xd2, 0x16, 0x2f}
+	// postgresCancelRequest is sent when a Postgres client requests cancel of
```

Comment:
```suggestion
	// postgresCancelRequest is sent when a Postgres client cancels
```

Reply:
Actually, "requests cancellation" would be more accurate (since actual cancellation it's not guaranteed) so I'll keep as-is but tweak the language a bit.